svtools/bin/bedpesort

```diff
@@ -6,12 +6,12 @@ if [ ! -z "$1" ]
 then
 	if [ ! -z "$2" ]
 	then
-		awk '$0~"^#" { print $0; next } { print $0 | "sort -k1,1V -k2,2n -k3,3n -k4,4V -k5,5n -k6,6n" }' $1 >$2
```

Comment:
This is the way is was originally.  I changed it to numeric in a pull
request a couple of weeks ago.   But then I was running into a random
error--it turned out to be unrelated, but I changed it back just in case
and forgot to fix it.

On Wed, Apr 13, 2016 at 11:41 AM, Dave Larson <notifications@github.com>
wrote:

> In svtools/bin/bedpesort
> <https://github.com/hall-lab/svtools/pull/101#discussion_r59582073>:
>
> > @@ -6,12 +6,12 @@ if [ ! -z "$1" ]
> >  then
> >  	if [ ! -z "$2" ]
> >  	then
> > -		awk '$0~"^#" { print $0; next } { print $0 | "sort -k1,1V -k2,2n -k3,3n -k4,4V -k5,5n -k6,6n" }' $1 >$2
>
> Why are the sorts in bedpesort being changes to non-numeric on the end
> coordinate of the second breakpoint?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/hall-lab/svtools/pull/101/files/860ebee48fa07aea0c0bea0635fd36e876f1461b#r59582073>
>